crates/forge/src/lib.rs

```diff
@@ -203,13 +223,19 @@ pub fn run(
 
     let mut tests_iterator = tests.into_iter();
 
+    let mut fuzzer = match runner_config.fuzzer_seed {
+        None => fuzzer::Random::new(),
+        Some(seed) => fuzzer::Random::from_seed(seed),
+    };
```

Comment:
Why it is created per file not per test case? number of runs should be per test case

Reply:
Actually it is one fuzzer per package. I agree, but then seeds could be different for each test case - do we want that? If we do I can do it this way and print the seed after each test case